src/Sarif/DelegatingStream.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    internal class DelegatingStream : Stream
```

Comment:
>DelegatingStream [](http://example.com/codeflow?start=19&length=16)

I see no delegating stream tests.

Adding unit tests for all new classes is an important quality standard on our team.

Reply:
Gabe added in iteration 7.